setup.py

```diff
@@ -1,10 +1,20 @@
 # coding: utf-8
+from os import rename
 from os.path import join as pathjoin, dirname
+import subprocess
 from setuptools import setup, find_packages
+from setuptools.command.build_ext import build_ext
 
 def read(*rnames):
     return open(pathjoin(dirname(__file__), *rnames)).read()
 
+class GitCloneScripts(build_ext):
+    """ Clones Lua Scripts and puts in expected directory """
+    def run(self):
+        subprocess.check_call(['git', 'clone', 'https://github.com/wanderu/bamboo-scripts'])
```

Comment:
`git clone` optionally takes a second argument for the target directory, so the `rename()` isn't necessary
```suggestion
        subprocess.check_call(['git', 'clone', 'https://github.com/wanderu/bamboo-scripts', 'wanderu/bamboo/scripts'])
```